src/mumble/mumble_pch.hpp

```diff
@@ -81,6 +81,9 @@
 #define interface struct
 #endif
 
+#ifndef _WIN32_WINNT
```

Comment:
Please only define for MinGW. MSVC has a default somwhere, so that works already. Let's not explicitly set it to Vista for MSVC.